velox/common/base/CheckedArithmetic.h

```diff
@@ -96,9 +96,7 @@ inline UnscaledShortDecimal checkedMultiply(
       __builtin_mul_overflow(a.unscaledValue(), b.unscaledValue(), &result);
   if (UNLIKELY(overflow)) {
```

Comment:
Do we not need `valueInRange` check here?

Reply:
This PR https://github.com/facebookincubator/velox/pull/2494/files#diff-ba972f0854a87c036d8b3f4e749e3936b0e74fbd0fa4f3c92a647e99d94f2c5bR136 covers it.